test/python/WMCore_t/WMRuntime_t/Scripts_t/Timestamps_t.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+"""
+_Timestamps_t_
+
+Tests timestamps metrics
+"""
+
+import unittest
+
+from WMCore.Configuration import Configuration
+from WMCore.WMRuntime.Scripts.Timestamps import addTimestampMetrics
+
+
+class Timestamps(unittest.TestCase):
+    """
+    Timestamps class provides unit tests for Timestamps module
+    """
+    def addTimestamps(self):
```

Comment:
This test is not executed. Please create it accordingly to the other tests, such as:
1) name the class with a test string, e.g.: TimestampsTests
2) override the setUp and tearDown methods
3) name unit tests method starting with the `test` string, otherwise they won't get executed.

Reply:
thanks for catching this up, I simply miss the python way of doing unit tests here. Now it is fixed.